geode-core/src/main/java/org/apache/geode/distributed/internal/InternalConfigurationPersistenceService.java

```diff
@@ -128,6 +128,7 @@
    * Name of the region which is used to store the configuration information
    */
   public static final String CONFIG_REGION_NAME = "_ConfigurationRegion";
+  public static final String CACHE_CONFIG_VERSION = "1.0";
```

Comment:
surely this does not need to be public or?

Reply:
Sure, this can be private